src/index.ts

```diff
@@ -1,3 +1,6 @@
 export * from './parser'
+import { ParserError, parse, tokenize, visit } from './parser'
 
 export type { ParseOptions } from './types'
+
+export default { ParserError, parse, tokenize, visit }
```

Comment:
after some testing, this was only needed because we were importing a default value in `prettier-plugin-solidity`

if we change:
```JavaScript
import parser from '@solidity-parser/parser';
```
to:
```JavaScript
import { parse as solidityParserParse, visit } from '@solidity-parser/parser';
```

It works perfectly.
Up to you whether you wanna keep this change or not.

Reply:
Does this mean that default imports won't work anymore?

Reply:
Ok, I checked this locally and it works fine both with CJS and ESM :+1:

Reply:
The UMD file also works fine with HTML. I think we can merge this.